FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,45 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+			case .success(let response):
```

Comment:
To give more context multiple assignment for response tuple can be used here.

In the given suggestion `data` will have type `Data` and `response` will be `HTTPURLResponse`

```suggestion
			case let .success((data, response)):
```